job-server/src/main/resources/application.conf

```diff
@@ -5,6 +5,7 @@ spark {
 
   # client or cluster deployment
   submit.deployMode = "client"
+  driver.supervise = false
```

Comment:
How about add comment so others can understand what it is... can refer to README for more details

Reply:
Makes sense. I will add a Readme file with details regarding this mode.